Library/Formula/libwebm.rb

```diff
@@ -0,0 +1,29 @@
+require 'formula'
+
+class Libwebm < Formula
+  desc "WebM container"
+  homepage 'http://www.webmproject.org/code/'
+  url "https://github.com/webmproject/libwebm/archive/libwebm-1.0.0.26.tar.gz"
+  sha256 "1e4034e03836ac974bb078b11383a10306d63879b350a34fdd575ab44b132222"
+
+  head "https://chromium.googlesource.com/webm/libwebm", :using => :git
+
+  head do  
+    depends_on "cmake" => :build
+  end
+
+  def install
+     if build.head?
+        mkdir "macbuild" do
+           system "cmake", "..", *std_cmake_args
+           system "make"
+        end
+     else
+     	system "make"
+     end
+     lib.install "libwebm.a"
```

Comment:
No, there is no ```make install```, not sure why. Forgot to add headers. Uploading the changes in a bit.